packages/actor-init-sparql/test/ActorInitSparql-test.ts

```diff
@@ -583,6 +584,13 @@ describe('ActorInitSparql', () => {
       });
     });
 
+    it('bindings() should collect all bindings until "end" event occurs on triples', async() => {
```

Comment:
The tests look good, but will probably have to be changed a bit after my suggestions.

Could you also add some tests for empty results and erroring results?